LabBuilder/LabBuilder.psm1

```diff
@@ -1438,6 +1438,120 @@ function Get-LabVMTemplates {
     Return $VMTemplates
 } # Get-LabVMTemplates
 ####################################################################################################
+####################################################################################################
+<#
+.SYNOPSIS
+   Gets an Array of TemplateDisks for a Lab configuration.
+.DESCRIPTION
+   Takes a provided Lab Configuration file and returns the list of Template Disks
+   that will be used to create the Virtual Machines in this lab. This list is usually passed to
+   Initialize-LabVMTemplateDisks.
+.PARAMETER Configuration
+   Contains the Lab Builder configuration object that was loaded by the Get-LabConfiguration object.
+.EXAMPLE
+   $VMTemplateDisks = Get-LabVMTemplateDisks -Config c:\mylab\config.xml 
+   Loads a Lab Builder configuration and pulls the array of TemplateDisks from it.
+.OUTPUTS
+   Returns an array of TemplateDisks.
+#>
+function Get-LabVMTemplateDisks {
+    [OutputType([System.Collections.Hashtable[]])]
+    [CmdLetBinding()]
+    param
+    (
+        [Parameter(
+            Mandatory,
+            Position=0)]
+        [ValidateNotNullOrEmpty()]
+        [XML] $Config
+    )
+
+    [System.Collections.Hashtable[]] $VMTemplateDisks = @()
+    [String] $VHDParentPath = $Config.labbuilderconfig.SelectNodes('settings').vhdparentpath
+   
+	Write-Verbose "VHD parent path = $VHDParentPath" 
```

Comment:
Don't worry - I used to do the exact same things, and still would but when contributing to the MSFT PowerShell repos they like things to be just so - so I've adopted their practices.